### PR TITLE
SKN-30 build user crud endpoints

### PR DESCRIPTION
**Pull request by PedroLabrador**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Skynet REST API",
   "main": "index.js",
   "scripts": {
-    "test": "npx jest --passWithNoTests",
+    "test": "jest --passWithNoTests --watch",
```
Comment (maacaro) [inline package.json:7]: @PedroLabrador this --watch will keep the build phase for ever, we might want to create a separate command for the pipelines let me change it

**Review by maacaro — CHANGES_REQUESTED** [review]
@PedroLabrador we should change a command on the packge json

File: package.json (hunk 1 above)
Comment (PedroLabrador) [reply package.json:7]: ok sure

File: src/config/sequelize.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+require('dotenv').config();
```
[the remark this answers is not in the document]
Comment (PedroLabrador) [reply src/config/sequelize.js:1]: to reload .env variables into the app

**Review by maacaro — COMMENTED** [review]
LGTM

**Pull request merged**